app.js

```diff
@@ -23,6 +23,14 @@ ApolloServer.prototype._createGraphQLServerOptions =
 
 ApolloServer.prototype.createGraphQLServerOptions = async function(req, res) {
   const options = await this._createGraphQLServerOptions(req, res);
+  logger.info(
```

Comment:
this log is pretty verbose... maybe set it to debug level so we can filter out when not needed.